src/treq/api.py

```diff
@@ -107,6 +107,12 @@ def request(method, url, **kwargs):
     :param bool unbuffered: Pass ``True`` to to disable response buffering.  By
         default treq buffers the entire response body in memory.
 
+    :param agent: Provide your own custom agent. Use this to override things
+                  like ``connectTimeout`` or ``BrowserLikePolicyForHTTPS``. By
+                  default, treq will create its own Agent with reasonable
+                  defaults.
+    :type agent: ``twisted.web.client.Agent``
```

Comment:
This should really be `twisted.web.iweb.IAgent`, [without the double backticks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists), which inhibit hyperlinking.

Reply:
Thanks, I've amended the commit with your suggestion and re-pushed